torch/_inductor/codegen/triton.py

```diff
@@ -2013,7 +2013,7 @@ def define_kernel(self, src_code, node_schedule):
             # not use BracesBuffer, so we have no good indicator of a C++ buffer atm.
             src_code = src_code.replace("#pragma CMT", "#")
 
-            basename, _, kernel_path = get_code_path(src_code, "py", extra="")
+            basename, _, kernel_path = get_path(code_hash(src_code), "py")
```

Comment:
We should consider landing some of the refactory bits first. I could set you up with ghstack if you want

Reply:
let's do that yeah